assets/dev/js/editor/container/container.js

```diff
@@ -116,6 +116,13 @@ export default class Container extends ArgsObject {
 	 */
 	panel;
 
+	/**
+	 * Controls placeholders.
+	 *
+	 * @type {{}}
+	 */
+	placeholders = {};
```

Comment:
This property is not relevant for all containers. It should only be applied to  the panel editor page container

Reply:
> This property is not relevant for all containers. It should only be applied to the panel editor page container

It appears that this change is fine with the controls themselves, but the dynamic tags throw some errors. Do you want to proceed with this approach?